src/components/collapsible.less

```diff
@@ -13,12 +13,10 @@
 }
 
 .Collapsible-header {
```

Comment:
The collapsible arrow is no longer aligned to the right, which I find looks not so good (especially when the title is omitted):

<img width="749" alt="Screen Shot 2022-08-31 at 17 21 49" src="https://user-images.githubusercontent.com/8189923/187716645-fb7d5a0a-561f-459c-a693-cb0a4f70f1b3.png">

Reply:
Nice catch! I haven't realized. It's fixed now.